include/pipettes/tests/test_motor_control.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <iostream>
+#include <span>
+
+#include "common/core/bit_utils.hpp"
+//#include "pipettes/firmware/motor_control.hpp"
+
+/*
+ * Motor Control and namespace.
```

Comment:
kind of a confusing block comment - maybe mostly just mention this is a test mock for the real thing?